internal/pkg/template/templates/workloads/partials/cf/alb.yml

```diff
@@ -1,50 +1,51 @@
-TargetGroup:
+{{- range $i, $rule := .ALB.Rules}}
+TargetGroup{{ if ne $i 0 }}{{ $i }}{{ end }}:
   Metadata:
     'aws:copilot:description': "A target group to connect the load balancer to your service"
   Type: AWS::ElasticLoadBalancingV2::TargetGroup
   Properties:
-    HealthCheckPath: {{.HTTPHealthCheck.HealthCheckPath}} # Default is '/'.
-    {{- if .HTTPHealthCheck.Port}}
-    HealthCheckPort: {{.HTTPHealthCheck.Port}} # Default is 'traffic-port'.
+    HealthCheckPath: {{$rule.HTTPHealthCheck.HealthCheckPath}} # Default is '/'.
+    {{- if $rule.HTTPHealthCheck.Port}}
+    HealthCheckPort: {{$rule.HTTPHealthCheck.Port}} # Default is 'traffic-port'.
     {{- end}}
-    {{- if .HTTPHealthCheck.SuccessCodes}}
+    {{- if $rule.HTTPHealthCheck.SuccessCodes}}
     Matcher:
-      HttpCode: {{.HTTPHealthCheck.SuccessCodes}}
+      HttpCode: {{$rule.HTTPHealthCheck.SuccessCodes}}
     {{- end}}
-    {{- if .HTTPHealthCheck.HealthyThreshold}}
-    HealthyThresholdCount: {{.HTTPHealthCheck.HealthyThreshold}}
+    {{- if $rule.HTTPHealthCheck.HealthyThreshold}}
+    HealthyThresholdCount: {{$rule.HTTPHealthCheck.HealthyThreshold}}
     {{- end}}
-    {{- if .HTTPHealthCheck.UnhealthyThreshold}}
-    UnhealthyThresholdCount: {{.HTTPHealthCheck.UnhealthyThreshold}}
+    {{- if $rule.HTTPHealthCheck.UnhealthyThreshold}}
+    UnhealthyThresholdCount: {{$rule.HTTPHealthCheck.UnhealthyThreshold}}
     {{- end}}
-    {{- if .HTTPHealthCheck.Interval}}
-    HealthCheckIntervalSeconds: {{.HTTPHealthCheck.Interval}}
+    {{- if $rule.HTTPHealthCheck.Interval}}
+    HealthCheckIntervalSeconds: {{$rule.HTTPHealthCheck.Interval}}
     {{- end}}
-    {{- if .HTTPHealthCheck.Timeout}}
-    HealthCheckTimeoutSeconds: {{.HTTPHealthCheck.Timeout}}
+    {{- if $rule.HTTPHealthCheck.Timeout}}
+    HealthCheckTimeoutSeconds: {{$rule.HTTPHealthCheck.Timeout}}
     {{- end}}
-    {{- if .HealthCheckProtocol}}
-    HealthCheckProtocol: {{.HealthCheckProtocol}}
+    {{- if $rule.HealthCheckProtocol}}
+    HealthCheckProtocol: {{$rule.HealthCheckProtocol}}
     {{- end}}
-    Port: !Ref TargetPort
-    {{- if .HTTPTargetContainer.IsHTTPS }}
+    Port: {{$rule.TargetPort}}
+    {{- if $rule.IsHTTPS }}
     Protocol: HTTPS
     {{- else }}
     Protocol: HTTP
     {{- end }}
-    {{- if .HTTPVersion}}
-    ProtocolVersion: {{.HTTPVersion}}
+    {{- if $rule.HTTPVersion}}
+    ProtocolVersion: {{$rule.HTTPVersion}}
     {{- end}}
     TargetGroupAttributes:
       - Key: deregistration_delay.timeout_seconds
-        Value: {{.DeregistrationDelay}} # ECS Default is 300; Copilot default is 60.
+        Value: {{$.DeregistrationDelay}} # ECS Default is 300; Copilot default is 60.
       - Key: stickiness.enabled
-        Value: !Ref Stickiness
+        Value: {{$rule.Stickiness}}
```

Comment:
does this render properly in CFN? because stickiness is a bool and not a string in the Go struct

Reply:
yes it does work fine because [Stickiness](https://github.com/aws/copilot-cli/pull/4501/files#diff-53f2b158bad1d10014e86bdd2b5b6ccc88acb52a641ccfce4a65c9fad47517cdR454) is a string in the Go struct.